Toric/Mathlib/CategoryTheory/WithTerminal/Cones.lean

```diff
@@ -0,0 +1,137 @@
+import Mathlib.CategoryTheory.Comma.Over.Basic
```

Comment:
This is a new file, right? Could you then add a copyright header? (so that eg I can upstream it to mathlib, if you don't do it yourself)